service/src/test/java/greencity/service/ubs/SuperAdminServiceImplTest.java

```diff
@@ -658,19 +658,31 @@ void editNewTariffSuccess() {
         verify(tariffsInfoRepository, times(2)).save(any());
     }
 
+    @Test
+    void addNewTariffThrowsExceptionWhenListOfLocationsIsEmptyTest() {
+        AddNewTariffDto dto = ModelUtils.getAddNewTariffDto();
+        when(courierRepository.findById(1L)).thenReturn(Optional.of(ModelUtils.getCourier()));
+        when(receivingStationRepository.findAllById(List.of(1L))).thenReturn(ModelUtils.getReceivingList());
+        when(locationRepository.findAllByIdAndRegionId(dto.getLocationIdList(),
+            dto.getRegionId())).thenReturn(Collections.emptyList());
+        Throwable t = assertThrows(NotFoundException.class,
+            () -> superAdminService.addNewTariff(dto, "35467585763t4sfgchjfuyetf"));
+        Assertions.assertEquals("List of locations can not be empty", t.getMessage());
```

Comment:
please add `verify`

Reply:
You add 3 mocks
```
        when(courierRepository.findById(1L)).thenReturn(Optional.of(ModelUtils.getCourier()));
        when(receivingStationRepository.findAllById(List.of(1L))).thenReturn(ModelUtils.getReceivingList());
        when(locationRepository.findAllByIdAndRegionId(dto.getLocationIdList(),
```
please verify that all of them were called with the expected parms and there were no more interactions with mooks

Reply:
done